src/main/java/frc/robot/utils/DashboardDataContainer.java

```diff
@@ -1,16 +1,21 @@
 package frc.robot.utils;
 
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.subsystems.mixer.SpinMixer;
 import frc.robot.utils.DashboardController;
 
 /**
- * DashboardDataContainer contains all the data to be viewed or put in the dashboard.  
+ * DashboardDataContainer contains all the data to be viewed or put in the
+ * dashboard.
  */
 public class DashboardDataContainer {
     private DashboardController dashboardController;
-    
+
     public DashboardDataContainer() {
-        dashboardController = new DashboardController();
 
+        dashboardController = new DashboardController();
+        SmartDashboard.putNumber("Mixer Power", 0.0);
```

Comment:
replace "Mixer Power" with "Mixer/Mixer Power".
This makes a subtable of mixer so shuffleboard is cleaner